### PR TITLE
Upgrade test

### PR DESCRIPTION
**Pull request by gabrielhartmann**
Re-add the `hello-world` upgrade test.  Got lost on promotion of the pods branch to master.

### CODE REVIEW

**Comment by nickbp** [comment]
Could also delete the now-duplicate copy in examples/

File: frameworks/helloworld/integration/tests/test_utils.py [hunk 1]
```diff
@@ -68,10 +71,13 @@ def success_predicate(response):
     return spin(fn, success_predicate)
 
 
-def install(additional_options = {}):
+def install(package_version=None, package_name=PACKAGE_NAME, additional_options = {}):
```
Comment (susanxhuynh) [inline frameworks/helloworld/integration/tests/test_utils.py]: You can remove the second param, "package_name". It's not used.

Comment (gabrielhartmann) [inline frameworks/helloworld/integration/tests/test_utils.py]: It is being used in logging and in the `install_package_and_wait` command.

Comment (NimaVaziri) [inline frameworks/helloworld/integration/tests/test_utils.py]: Looks like the default value of the argument is being used (`PACKAGE_NAME`), and not any other value that could be passed in via that parameter.

**Pull request merged**